src/components/grid-aware/Footer/Footer.module.css

```diff
@@ -0,0 +1,257 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto 0;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-rows: 120px 380px 78px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    text-align: center;
+  }
+
+  .gridParent {
+    column-gap: var(--grid-column-gap);
+    display: grid;
+    grid-template-rows: 190px auto auto;
+  }
+}
+
+/* Top Row */
+
+.gridTopRow {
+  align-content: center;
+  background-color: var(--color-black);
+  display: flex;
+  grid-column: 1 / span 12;
+  grid-row: 1;
+  justify-content: center;
+}
+
+.subscribeContainer {
+  align-items: baseline;
+  align-self: center;
+  display: flex;
+  justify-content: center;
```

Comment:
I think you have a lot of unnecessary layering, flex-boxing, and centering going on, not just in this element but also in the other elements in this top row. I think this could be cut down significantly while still implementing the style.

To avoid having to do vertical centering, you can just add top padding and bottom padding to the row itself. This also makes your component more flexible because you won't have to hardcode a fixed height, and the height becomes more fluid depending on the height of the actual content.

Reply:
ok